resources/web/docs_js/components/console_widget.jsx

```diff
@@ -0,0 +1,96 @@
+import * as utils from "../utils.js";
```

Comment:
Just being picky about consistency, but sometimes you import with the `.js` extension on and sometimes you don't.

Reply:
Totally, need to drop js